Source/Util/Localization.cpp

```diff
@@ -46,6 +50,11 @@ void initLocale() {
         localeCurrent = "";
         saveLocale(localeCurrent);
     }
+
+#ifdef GPX
+    localeCurrent = gpx()->sys()->getLanguage() == "ru" ? "russian" : "english";
```

Comment:
Maybe is better to put this before the first if (localeCurrent.empty()) since GPX knows the language and we dont need to try guessing by reading .ini files or settings?